markdown/org/docs/patterns/otis/instructions/en.md

```diff
@@ -0,0 +1,88 @@
+---
+title: "Otis, the simple onesie: Sewing Instructions"
+---
+
+<Note>
+
+You should use a stretch stitch on your sewing machine. Small zig-zag stitches work really well. The side seams can also be done with a serger.
+
+</Note>
+
+## Step 1: Bindings
+
+### Two options
+
+There are two ways to do the bindings. You can use it like double-folded bias binding, or just fold it in two and stitch that to the pieces. The double-folding will look nicer and more finished, but is a lot more involved.
+
+#### Double-folded
+
+  - Fold both bindings along the long side in two, wrong sides together, and give it a good press.
+  - Mark the middle of each of the bindings and of the front and back parts. 
```

Comment:
I notice that there is a notch on the neck bindings, but the notch isn't mentioned in the instructions. Does the notch get used in this step?